README.md

```diff
@@ -411,7 +417,7 @@ module.exports = function(api) {
 {
   "jest": {
     "transform": {
-      "^.+\\.(ts|html)$": "<rootDir>/node_modules/jest-preset-angular/preprocessor.js",
+      "^.+\\.(ts|html)$": "ts-jest",
```

Comment:
Not sure about this, does this apply for a config with babel + ts-jest? Surely the referenced `preprocessor.js` would not work anymore.

Reply:
I remember when using with babel, people often specify pattern to use babel here to deal with js and pattern ts with ts-jest at this place, here is a [reference](https://stackoverflow.com/questions/52173815/ts-jest-does-not-recognize-es6-imports)

Reply:
Ok cool, so if I understand it properly, forwarding only `*.ts` to `ts-jest` should do the trick here, as done in the example. Do you agree?

Reply:
ya I think that should work. In the past we tried to override ts-jest preprocessor so that was why we needed to point to our `preprocessor.js`. Since now we don't have that anymore so we can use `ts-jest` and `babel-jest` like the example.

Reply:
and I think we should think about a way to opt-in `ts-jest` for `*ts.` files instead of forcing users to use it. Now babel has plugin for const enum as well, so when using babel the users only lose the type-checking functionality of `tsc`. 

In my opinion, we should give a free choice for users related to `babel` or `ts-jest` they want to use for `*.ts` files

Reply:
Ok, so an alternate setup with a babel would be great! Maybe running the example project twice?

That's a task for a different issue though.

Reply:
👍

Reply:
Opened #308